scripts/release.py

```diff
@@ -22,9 +23,7 @@ def announce(version):
 
     contributors = set(stdout.splitlines())
 
-    template_name = (
-        "release.minor.rst" if version.endswith(".0") else "release.patch.rst"
-    )
+    template_name = "release.minor.rst" if version.minor == 0 else "release.patch.rst"
```

Comment:
Have you seen my suggestions:

- In https://github.com/pytest-dev/pytest/issues/8820#issuecomment-870443249 regarding adding a template for prereleases
- In https://github.com/pytest-dev/pytest/pull/8814#discussion_r660084157 regarding passing through this information from `prepare-release-pr.py` rather than trying to infer it again from the version number

?

Reply:
Missed it before, will check the details, thanks